pkg/release/repository.go

```diff
@@ -108,6 +108,10 @@ func (r *Repo) CheckState(expOrg, expRepo, expBranch string) error {
 	if branch != expBranch {
 		return errors.Errorf("branch %q expected but got %q", expBranch, branch)
 	}
+	if nomock && !(expOrg == "kubernetes" && expRepo == "release" && expBranch == "master") {
```

Comment:
Please use the constants there:
https://github.com/kubernetes/release/blob/master/pkg/release/release.go#L52-L54

Reply:
Done. Could you help to review it? Thanks!